tests/test_cli_compile.py

```diff
@@ -2,16 +2,21 @@
 
 import hashlib
 import os
+import pathlib
 import shutil
 import subprocess
 import sys
 from textwrap import dedent
 from unittest import mock
+from unittest.mock import MagicMock
 
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
Nit: `MokeyPatch` is available through the public API since pytest v6.2.0. I don't mind if it's not fixed right now. Just thought I'd post this as an educational comment.

_(https://docs.pytest.org/en/7.1.x/changelog.html#id142)_

Reply:
Thanks! I'm marking this as resolved.